actions/imanage/functions/imanage/1.0/index.js

```diff
@@ -0,0 +1,221 @@
+import lodash from "lodash";
+
+const MODEL_LOOKUP = {
+    getAllDocuments: {
+        url: "https://cloudimanage.com/work/api/v2/customers/{customerId}/documents",
+        method: "GET",
+        processResponse: (respose) => {
+            return respose.data.results
+        },
+        returnFormat: (results) => {
+            return {
+                results: results,
+                totalCount: results.length, 
+            }
+        }
+    },
+    getAllWorkspaces: {
+        url: "https://cloudimanage.com/work/api/v2/customers/{customerId}/workspaces",
+        method: "GET",
+        processResponse: (respose) => {
+            return respose.data.results
+        },
+        returnFormat: (results) => {
+            return {
+                results: results,
+                totalCount: results.length, 
+            }
+        }
+    },
+    getAllDocumentNVP: {
+        url: "https://cloudimanage.com/work/api/v2/customers/{customerId}/libraries/{libraryName}/documents/{documentId}/name-value-pairs",
+        method: "GET",
+        processResponse: (respose) => {
+            return respose.data
+        },
+        returnFormat: (results) => {
+            return {
+                results: results,
+                totalCount: results.length, 
+            }
+        }
+    },
+    getAllFolders: {
+        url: "https://cloudimanage.com/work/api/v2/customers/{customerId}/folders",
+        method: "GET",
+        processResponse: (respose) => {
+            return respose.data
+        },
+        returnFormat: (results) => {
+            return {
+                results: results,
+                totalCount: results.length, 
+            }
+        }
+    },
+    getAllFolderChildren: {
+        url: "https://cloudimanage.com/work/api/v2/customers/{customerId}/libraries/{libraryName}/folders/{folderId}/children",
```

Comment:
Maybe good to split the host and common part in the path into a seperate var and reuse this when setting the url